adsingestp/parsers/crossref.py

```diff
@@ -83,7 +83,10 @@ def _parse_pub(self):
         issns = []
         for i in issn_all:
             if i.get_text() and re_issn.match(i.get_text()):
-                issns.append((i["media_type"], i.get_text()))
+                if i.has_attr("media_type"):
```

Comment:
I think `i.get("media_type")` works the same way as this (check that), and that's what we've been using elsewhere. Use this syntax instead, for consistency